unit-9/exercise-2.js

```diff
@@ -0,0 +1,11 @@
+const ageclass = (age) => {
+	if( age < 16 ) {
+    return 'Bạn đã quá tuổi để vào lớp 10'
+	}
```

Comment:
Tab vẫn rất lộn xộn thế này.
Nhìn kỹ cách viết các dấu cách:
```suggestion
if (age < 16) {
    return 'Bạn đã quá tuổi để vào lớp 10'
}
```